e2e_playwright/st_dialog_test.py

```diff
@@ -81,9 +81,9 @@ def test_dialog_dismisses_properly(app: Page):
 def test_dialog_reopens_properly_after_dismiss(app: Page):
     """Test that dialog reopens after dismiss."""
     # open and close the dialog multiple times
-    for i in range(0, 5):
+    for _ in range(0, 5):
         # don't click indefinitely fast to give the dialog time to set the state
-        app.wait_for_timeout(100)
+        app.wait_for_timeout(250)
```

Comment:
I use the opportunity to increase the timeout, as I have seen that the test occasionally has been flaky.

Reply:
btw. I also decreased this range to 5 instead of 10 since I had seen that it had a quite long runtime a couple of times stretching out the total CI time. But 5 iterations are probably fine here as well, or?

Reply:
yep sounds good 👍